browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3813,6 +3813,15 @@ If you don't accept this request, VPN will not reconnect and your internet conne
       If your current service provider is insecure, Brave uses a secure alternative DNS provider
     </message>
     <!-- End of Secure DNS Settings -->
+    <message name="IDS_I_AGREE_ACTION" desc="CTA text for tos rewards panel">
+      I agree
+    </message>
+    <message name="IDS_REWARDS_TOS_TITLE" desc="CTA text for tos rewards panel">
+      Updated Terms of Service
+    </message>
+    <message name="IDS_REWARDS_TOS_TEXT" desc="CTA text for tos rewards panel">
+      We’ve updated the Terms of Service for Brave Rewards. We’ve made these updates to clarify our terms and ensure they cover new features. If you continue to use Brave Rewards, you are agreeing to the updated Terms of Service. If you prefer not to agree, you have the option to <ph name="RESET_START">&lt;reset&gt;</ph>reset<ph name="RESET_END">&lt;/reset&gt;</ph> Brave Rewards, and this will disable the feature.\n\nSee <ph name="TOS_START">&lt;tos&gt;</ph>Brave Rewards Terms of Service<ph name="TOS_END">&lt;/tos&gt;</ph>
+    </message>
```

Comment:
Can we not keep the strings the same as in https://github.com/brave/brave-core/pull/22367/files?file-filters%5B%5D=.grdp&show-deleted-files=true&show-viewed-files=true?

Reply:
yes we can. i will update from my end. Thank you @mkarolin

Reply:
@mkarolin actually i used https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/android/strings/android_chrome_strings.grd;l=1313?q=android_chrome_strings.grd&ss=chromium as a reference. if i use strings as suggested we need to add html tags instead of android specific spans.

Reply:
I think it's ok if we have it duplicated here but with the extra tags. As long as the rest of the text matches we wouldn't have to retranslate.